src/main/mdo/changes.mdo

```diff
@@ -227,8 +227,8 @@ under the License.
         <field xml.attribute="true">
           <name>dev</name>
           <version>1.0.0+</version>
-          <required>true</required>
           <type>String</type>
+          <defaultValue/>
```

Comment:
Isn't this redundant?

Reply:
with default value as empty string I have in report `-` instead of nbps;